mathics/core/definitions.py

```diff
@@ -112,17 +113,16 @@ def __init__(
             "System`",
             "Global`",
         )
+
+        # importing format loads populates Symbols to PrintForms and OutputForms
+        # Rocky: this smells of something not quite right in terms of modularity.
+        import mathics.format  # noqa
```

Comment:
Interesting. In which line this happens?

Reply:
It is not an error, it is just that the output won't get tagged (which is the point of the change) since `result.form` will be `None` because nothing in `output_forms`: 

```
+                if head in output_forms:
+                    form = self.definitions.shorten_name(head.name)
```